pom.xml

```diff
@@ -25,8 +25,7 @@
     </parent>
 
     <artifactId>org.apache.sling.superimposing</artifactId>
-    <version>0.2.1-SNAPSHOT</version>
-    <packaging>bundle</packaging>
+    <version>1.0.0</version>
```

Comment:
Please revert the version change, we change to non snapshots only when releasing

Reply:
Reverted to 0.2.1-SNAPSHOT